tools/session-ui/server.rb

```diff
@@ -1,3 +1,8 @@
+<<<<<<< HEAD
```

Comment:
Please, clean up this file, it looks like you committed an intermediate a file that had a conflicts during merge.

Reply:
The Server class inside server.rb has been merged into Intermediate module inside backend.rb. This way, the server can act as mixin and can be instantiated within msfconsole.

Reply:
That is fine, however this file is still here and contains artifacts from a conflicted git merge.

`<<<<<<< HEAD` is the beginning of a failed merge block.  It sounds like you plan to remove this completely. If so just remove it now, otherwise it needs to be cleaned up.

Reply:
Fixed this issue